qmd/protocols.qmd

```diff
@@ -0,0 +1,162 @@
+
+![Figure I: Protocol Relationships - PEG](./assets/protocols/PEG_relationships-1.png)
+
+To solve this issue, we again lean on the sub-protocols to better define the separate centrifugation protocols, so that they can be referenced unambiguously in the PEG protocol. See you this is done below in Figure J.
+
+![Figure J: Sub-Protocols Into Protocol Steps - PEG](./assets/protocols/PEG_relationships-2.png)
+
+Again, the final details on the protocols are stored in the protocols table (Figure K). This includes summaries of the protocols, links to references, and version information.
+
+![Figure K: Protocols Table - PEG](./assets/protocols/PEG_protocols.png)
+
+## Example Protocol: Methods Text to ODM Data - RNA Extraction
+
+Continuing with this example from the (Delatolla Research Group paper)[https://www.sciencedirect.com/science/article/pii/S0043135420310952?via%3Dihub], we will now look at how the RNA Extraction protocol was described in the methods section of the paper (Figure L).
+
+![Figure L: Methods Section Breakdown - RNA Extraction](./assets/protocols/extraction_methods.png)
+
+Here again we see that this information describes one protocol for all samples. More sepcifically, because the RNA extraction uses a kit, the process can actually be described as a single protocol step (Figure M). 
+
+![Figure M: Methods Into Protocol Steps - RNA Extraction](./assets/protocols/extraction_steps.png)
+
+There was a lot more information in the methods section description of this step than just the kit they used. For instances like this, there is a `summary` column in the protocol steps table to record alterations that are made to a protocol step. For these example images we have used abridged tables to simplify the data-flow, but the actual protocol steps table includes other headers for references, summaries, links, and other important metadata. 
+
+Because this process is summarized as a single step, it has no relationships to manage, nor and container protocol. The RNA extraction protocol step is self-contained.
+
+## Example Protocol: Methods Text to ODM Data - PCR Quantification
+
+Still using the methods section from the (Delatolla Research Group paper)[https://www.sciencedirect.com/science/article/pii/S0043135420310952?via%3Dihub], we will now look at how to record the PCR quantification protocol data. As with the other examples, we will start with the methods section of the paper (Figure N).
+
+![Figure N: Methods Section Breakdown - PCR Quantification](./assets/protocols/quantification_methods.png)
+The text describes a single protocol for all samples, however it's clear that the process is quite complicated and will likely need sub-protocols. We can start by breaking this section down into protocol steps and populating the protocol steps table (Figure O).
+
+![Figure O: Methods Into Protocol Steps - PCR Quantification](./assets/protocols/quantification_steps.png)
```

Comment:
- The ends of the arrows around "Primer mix protocol" are a bit wonky. 
- Great job breaking down such a complex protocol! Glad to see that the structure seems to hold up.
- The Primer mix protocol specifies the PCR method and is before the PCR Thermocycling protocol. But doesn't the PCR thermocycling protocol also specify the PCR method?
- There are a lot of arrows, and they don't all represent the same thing. To lighten the visual burden, could this figure be broken down into 2? One with only the relationships between steps, and another with the step groups (sub-protocols)?